src/register-tools.ts

```diff
@@ -1,9 +1,16 @@
 // import all tools so they are registered
 import "./models/tools/unknown-content";
 import "./models/tools/placeholder/placeholder-registration";
-import "./models/tools/drawing/drawing-registration";
-import "./models/tools/text/text-registration";
-import "./models/tools/table/table-registration";
-import "./models/tools/image/image-registration";
-import "./models/tools/geometry/geometry-registration";
-import "./plugins/diagram-viewer/diagram-registration";
+
+const gToolRegistration: Record<string, () => void> = {
+  "Diagram": () => import(/* webpackChunkName: "Diagram" */"./plugins/diagram-viewer/diagram-registration"),
```

Comment:
Should we add the Diagram tool to the tool-tiles.test.ts, so this line of code is covered?